test/chemical_model.jl

```diff
@@ -0,0 +1,20 @@
+using DiffEqSensitivity, OrdinaryDiffEq, ParameterizedFunctions
```

Comment:
This should go to diffeqproblemlibrary (with the analytical solution)

Reply:
Will move it once the method is ready, currently using it as quick tests for implementation.